contracts/GovHub.sol

```diff
@@ -19,7 +19,7 @@ contract GovHub is System, IApplication {
 
     event failReasonWithStr(string message);
     event failReasonWithBytes(bytes message);
-    event paramChange(string key, bytes value);
+    event paramChange(string key, bytes value);  // @dev deprecated
```

Comment:
simplify the `imports` here, like `RLPDecode.sol` is no longer used.

Reply:
removed